### PR TITLE
Feature/line management

### PR DESCRIPTION
**Pull request by harry-m**
A couple of new sections. One on line management in general, and another on related DM responsibilities.

For discussion.

### CODE REVIEW

**Comment by dgmstuart** [comment]
LGTM 👍

File: playbook.md [hunk 1]
```diff
@@ -813,6 +813,28 @@ We prefer to assume that everyone wants to improve, and will do so given the tim
...
+* Approving training and conference attendance
+* Caring for and nurturing a team to ensure it's effective
+
+These responsibilities are often given to people in addition to a primary role, such as being a developer or researcher.
+
+At dxw, we don't think this is an effective way to work. We want experienced delivery staff to spend their time and effort on the things that they were passionate enough about to learn to be good at. For most, this includes some of the responsibilities above, but it's very rare that it includes all of them.
+
+So, at dxw, the role of line manager is split up, so that each aspect is looked after by the best person for the job:
+
+* Delivery managers: performance, allocation of work, caring for the team
```
Comment (staceydxw) [inline playbook.md]: Could we have a conversation about where the 'line' is between managing performance and being responsible for handling disciplinaries and grievances? This is the main thing I'm confused about.

File: playbook.md [hunk 2]
```diff
@@ -813,6 +813,28 @@ We prefer to assume that everyone wants to improve, and will do so given the tim
...
+* Business operations: Holiday, sickness, employment issues
+* MD: Disciplinaries and grievances, training and conferences
```
Comment (staceydxw) [inline playbook.md]: When we say 'training and conferences', what does this mean? Is this helping find suitable opportunities for staff to develop their skills or learn a new thing, or just booking them? I don't feel like there's anyone at dxw that has ownership of learning & development as a thing (and this should be a thing)

File: playbook.md [hunk 3]
```diff
@@ -813,6 +813,28 @@ We prefer to assume that everyone wants to improve, and will do so given the tim
...
+* Any appropriate colleague: supervision of work, advice
```
Comment (staceydxw) [inline playbook.md]: Could we define what 'supervision of work' means in this context. Is it code reviews or something else?

File: playbook.md (hunk 2 above)
Comment (harry-m) [inline playbook.md]: I think in that line I meant talking about them and signing off the money. Booking them would be bizops.

File: playbook.md (hunk 3 above)
Comment (harry-m) [inline playbook.md]: Yep - code reviews, showing someone through a feature you made, that sort of thing. I'll reword.

File: playbook.md (hunk 1 above)
Comment (harry-m) [inline playbook.md]: Do we still need to talk about this?

**Comment by CristinaRO** [comment]
This PR hasn't had any discussion in over a year, so we've decided to close it (Dev forum Oct 19 2018).

If the subject still needs discussion and Playbook changes, please consider writing a Trello card in the [dxw Fridays Reloaded](https://trello.com/b/Hab7frYo/dxw-fridays-reloaded) board, so we can track its activity.

**Pull request closed without merging**